ted/src/main/java/frc/robot/commands/UseVisionForPositionCommand.java

```diff
@@ -0,0 +1,67 @@
+// ************************************************************
```

Comment:
Seems like this is not in use in the codebase in this PR.  When will it be used?  Should it be added then?

Reply:
this command allows us to update our odometry with vision, so if we wanted to do this in auto or for yaw calculations it would be useful.

Reply:
this doesn't yet work.  you should remove this file to a new branch for future testing.